source/framework/core/inc/TRestEventViewer.h

```diff
@@ -19,20 +19,19 @@
 #ifndef RestCore_TRestEventViewer
 #define RestCore_TRestEventViewer
 
+#include <TGButton.h>
+#include <TGLabel.h>
+#include <TGTextEntry.h>
 #include <TGeoManager.h>
 #include <TObject.h>
+#include <TRestEvent.h>
 
 #include <iostream>
 
-#include "TGButton.h"
-#include "TGLabel.h"
-#include "TGTextEntry.h"
-#include "TRestEvent.h"
-
 class TRestBrowser;
 class TRestEventViewer {
    protected:
-    TGeoManager* fGeometry = 0;     //!
+    TGeoManager* fGeometry = 0;  //!
```

Comment:
TGeoManager* fGeometry = nullptr;

Reply:
I will remove this instead (`TGeoManager* fGeometry = 0` -> `TGeoManager* fGeometry;`) since if I am not wrong there is no reason to initialize them to nullptr besides readability right?

Reply:
Looking at source/framework/core/src/TRestEventViewer.cxx looks like you can call TRestEveEventViewer::SetGeometry(TGeoManager* geo) with fGeometry un-initialized, there is actually a protection about that. I would suggest to keep the initialization as nullptr, shouldn't be a problem.

Reply:
Allways better to initialize